src/pages/gateway/create-mesh.md

```diff
@@ -321,6 +349,26 @@ To access the gateway and perform GraphQL queries, you need to provide an API Ke
 
 You can return to the Project Overview page whenever you need to retrieve your API Key.
 
+## Stage to production
+
+To move a mesh in the `stage` workspace to the `production` workspace:
+
+1. [Retrieve](#retrieve-a-previously-created-meshid) your previously created mesh by running the following command with the desired project and workspace selected.
+
+  ```bash
+  aio api-mesh:get
+  ```
+
+1. Copy and paste the retrieved mesh into a new `.json` file and save it.
+
+1. Select the `production` workspace or [select another project and workspace](#select-a-project-or-workspace) combination for your destination.
```

Comment:
The heading here is "Stage to production", so the "or select another project and workspace" seems out of place. I suggest rewriting this item to something like

1. Use the [`aio console:workspace:select`](#select-a-project-or-workspace) command to switch to your Production workspace.

That, or rename the section to "Move the mesh to another project".